<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphEditorForm</class>
 <widget class="QWidget" name="GraphEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ОСНОВНЫЕ СТИЛИ */
QWidget {
    background-color: #1f1f1f;
    color: #c0c0c0;
    font-family: &quot;Segoe UI&quot;, Arial, sans-serif;
    border: 0px;
}

/*  ГРУП БОКСЫ */
QGroupBox {
    border: 1px solid #1f4f4d;
    border-radius: 4px;
    margin-top: 10px;
    padding-top: 15px;
    background-color: #1a1a1a;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px;
    color: #d0d0d0;
}

/*  КНОПКИ */
QPushButton {
    background-color: #2a2a2a;
    color: #d0d0d0;
    border: 2px solid #1f4f4d;
    border-radius: 5px;
    padding: 3px 4px;
    min-width: 100px;
}

QPushButton:disabled {
    background-color: #2d2d2d;
    color: #6a6a6a;
    border: 1px solid #3a3a3a;
}

QPushButton:hover {
    background-color: #2d2d2d;
    border: 2px solid #30605e;
    color: #bbb;
}

QPushButton:pressed {
	background-color: #383838;
    border: 2px solid #777;
	padding-top: 6px;
    padding-bottom: 4px;
}

/* СЛАЙДЕРЫ */
QScrollBar {
    border: none;
    background: #0d0d0d;
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
	background-color: #2a2a2a;
    border: 2px solid #1f4f4d;
    min-height: 20px;
    border-radius: 4px;
    margin: 2px;
}

QScrollBar::handle:horizontal {
	background-color: #2a2a2a;
    border: 2px solid #1f4f4d;
    min-width: 20px;
    border-radius: 4px;
    margin: 2px;
}

QScrollBar::handle:hover {
    background-color: #2d2d2d;
    border: 2px solid #30605e;
    color: #bbb;
}

QScrollBar::handle:pressed {
    background-color: #383838;
    border: 2px solid #777;
	padding-top: 6px;
    padding-bottom: 4px;
}

QScrollBar::add-line, QScrollBar::sub-line {
    background: transparent;
    height: 0px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::up-arrow, QScrollBar::down-arrow {
    width: 0px;
    height: 0px;
}

QScrollBar::add-page, QScrollBar::sub-page {
    background: none;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="graphScene_groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Отображение графа</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="ObjectScene" name="graphScene" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="graphProps_toggle_pushButton">
        <property name="text">
         <string>Кнопка тоггла свойств графа</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="graphProps_groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Свойства графа</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="graphProps_common_groupBox">
        <property name="title">
         <string>Общие</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTableView" name="propertyCommon_tableView"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="graphProps_user_groupBox">
        <property name="title">
         <string>Пользовательские</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTableView" name="propertyUser_tableView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="propertyAdd_pushButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Добавить</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="propertySave_pushButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Сохранить</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="propertySaveAs_pushButton">
               <property name="text">
                <string>Сохранить как...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QPushButton" name="propertyRemove_pushButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Удалить</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="propertyLoad_pushButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Откатить</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="propertyLoadAs_pushButton">
               <property name="text">
                <string>Открыть граф...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ObjectScene</class>
   <extends>QWidget</extends>
   <header>src/GUI/ObjectScene/objectscene.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
